packages/main/src/plugin/telemetry/telemetry.spec.ts

```diff
@@ -368,3 +376,60 @@ describe('aggregateTrack', () => {
     expect(pending?.properties).toEqual([{ foo: 1 }, { bar: 2 }]);
   });
 });
+
+function safeSubdir(subdir?: string): string {
+  if (!subdir) return '';
+  return process.platform === 'win32' ? subdir.replace(/:/g, '_') : subdir;
+}
+
+async function prepareCerts(tmpHome: string, subdir?: string, files: Array<[string, string]> = []): Promise<void> {
+  const base = path.join(tmpHome, '.config', 'containers', 'certs.d', safeSubdir(subdir));
+  await fs.mkdir(base, { recursive: true });
+  await Promise.all(files.map(([name, content]) => fs.writeFile(path.join(base, name), content)));
+}
+
+describe('hasCustomCertificates', () => {
+  let tmpHome: string;
+  let prevHome: string | undefined;
+
+  beforeEach(async () => {
+    tmpHome = await fs.mkdtemp(path.join(os.tmpdir(), 'pd-certs-'));
+    prevHome = process.env['HOME'];
+    process.env['HOME'] = tmpHome;
+    process.env['XDG_CONFIG_HOME'] = path.join(tmpHome, '.config');
+  });
+
+  afterEach(async () => {
+    process.env['HOME'] = prevHome;
+    delete process.env['XDG_CONFIG_HOME'];
+    await fs.rm(tmpHome, { recursive: true, force: true });
+  });
+
+  const scenarios: Array<{
+    name: string;
+    setup: (tmp: string) => Promise<void>;
+    expected: boolean;
+  }> = [
+    { name: 'no cert dirs', setup: async (): Promise<void> => {}, expected: false },
+    {
+      name: 'certs.d exists but no subdirs',
+      setup: tmp => prepareCerts(tmp, undefined, [['note.txt', 'hello']]),
+      expected: false,
+    },
+    {
+      name: 'subdir without cert files',
+      setup: tmp => prepareCerts(tmp, 'example.com:5000', [['readme', 'no certs here']]),
+      expected: false,
+    },
+    {
+      name: 'ca.crt present in subdir',
+      setup: tmp => prepareCerts(tmp, 'registry.test:5001', [['ca.crt', '-----BEGIN CERTIFICATE-----\nDUMMY\n']]),
+      expected: true,
+    },
+  ];
+
+  test.each(scenarios)('$name', async ({ setup, expected }) => {
```

Comment:
It seems that this test is failing on windows, can anyone with this OS help to investigate?